PROTOCOL.md

```diff
@@ -11,10 +11,9 @@ The WebSocket sub-protocol for this specification is: `graphql-transport-ws`.
 
 Messages are represented through the JSON structure and are stringified before being sent over the network. They are bidirectional, meaning both the server and the client must conform to the specified message structure.
 
-**All** messages contain the `type` field outlining the action this message describes. Depending on the type, the message can contain two more _optional_ fields:
+**All** messages contain the `type` field outlining the action this message describes. Depending on the type, the message can contain the _optional_ field `payload` holding the extra "payload" information to go with the specific message type.
```

Comment:
The `payload` is also not always optional. I would just leave out the whole next sentence.

```suggestion
**All** messages contain the `type` field outlining the action this message describes.
```